src/configuration/catalog/catalog.md

```diff
@@ -196,34 +196,34 @@ When Live Search is installed, Catalog Search includes the following configurati
 
 |Field|[Scope]({% link configuration/scope.md %})|Description|
 |--- |--- |--- |
-|Minimal Query Length|Store View|The minimum number of characters allowed in a catalog search. The value set for this option must be compatible with the corresponding range set in your Elasticsearch search engine configurations. For example, if you set this value to `2` in Adobe Commerce, update the value in your search engine.|
-|Maximum Query Length|Store View|The maximum number of characters allowed in a catalog search. The value set for this option must be compatible with the corresponding range set in your Elasticsearch search engine configurations. For example, if you set this value to 300 in Adobe Commerce, update the value in your search engine.|
+|Minimal Query Length|Store View|The minimum number of characters allowed in a catalog search. The value set for this option must be compatible with the corresponding range set in your Elasticsearch/OpenSearch search engine configurations. For example, if you set this value to `2` in Adobe Commerce, update the value in your search engine.|
+|Maximum Query Length|Store View|The maximum number of characters allowed in a catalog search. The value set for this option must be compatible with the corresponding range set in your Elasticsearch/OpenSearch search engine configurations. For example, if you set this value to 300 in Adobe Commerce, update the value in your search engine.|
 |Number of top search results to cache|Store View|The number of popular search terms and results to cache for faster responses. Entering a value of `0` caches all search terms and results when entered a second time. Default value: 100|
 |Autocomplete Limit|Store View|Determines the maximum number of lines available in the [storefront popover] page. The default value can be changed when Live Search is installed, and updated later by changing this configuration setting. Default value: 8|
 
-### Adobe Commerce with Elasticsearch
+### Adobe Commerce with Elasticsearch/OpenSearch
 
-Native Adobe Commerce with Elasticsearch includes the following configuration settings:
+Native Adobe Commerce with Elasticsearch/OpenSearch includes the following configuration settings:
 
 ![]({% link configuration/catalog/assets/catalog-search-elasticsearch.png %}){: .zoom}
```

Comment:
Will we need an updated screen capture here?

Reply:
I've added a new screenshot with the OpenSearch settings